pkgs/unified_analytics/lib/src/analytics.dart

```diff
@@ -496,22 +487,25 @@ class AnalyticsImpl implements Analytics {
 
   @override
   void clientShowedMessage() {
+    // Check the tool needs to be added to the config file
     if (!_configHandler.parsedTools.containsKey(tool.label)) {
       _configHandler.addTool(
         tool: tool.label,
         versionNumber: toolsMessageVersion,
       );
-      _showMessage = true;
+      _showMessage = false;
     }
+
+    // When the tool already exists but the consent message version
+    // has been updated
     if (_configHandler.parsedTools[tool.label]!.versionNumber <
         toolsMessageVersion) {
       _configHandler.incrementToolVersion(
         tool: tool.label,
         newVersionNumber: toolsMessageVersion,
       );
-      _showMessage = true;
+      _showMessage = false;
```

Comment:
should _showMessage be set to false in the body of clientShowedMessage instead of in the blocks of these if statements?

Reply:
Yep it definitely can! Just made that change